samples/OpenIdConnect.AzureAdSample/OpenIdConnect.AzureAdSample.csproj

```diff
@@ -20,7 +20,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="$(IdentityModelActiveDirectoryVersion)" />
-    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="$(AspNetCoreVersion)" />
```

Comment:
Looks like there's a change \ bug in `GenerateRestoreGraphFile` where it only lists one CLI tool reference per solution. I'm going to try and get a smallish repro app and file a bug with NuGet. In the meanwhiile, this should avoid build issues since pinning is messed up now.